### ISSUE TITLE
Windows shell writes the output in the wrong encoding

### ISSUE THREAD

**Issue by kotauskas** [livioribeiro/cargo-readme#49]
I'm using PowerShell to run the command — the resulting file is in little-endian UTF-16 rather than UTF-8, since the output redirection and file I/O is done entirely by the shell. Obviously, Cargo rejects such a `README`. The simplest workaround is to add a new command-line argument, `-o`, which would accept a filename, just like, for example, `mingw-w64` GCC does to mitigate the fact that Windows standard I/O steams have an encoding.

Calling `cargo readme` without arguments then should result in an error (`error: no output destination supplied`), and `cargo readme -` should imitate the old behavior.

Also, (I think that's related to this issue, might be wrong), the — (`EM DASH`) symbol in my doc comment got converted to gibberish from my current code page (cp1251), even after I re-encoded the result into valid UTF-8.

**Comment by udoprog** [comment]
Another interesting issue is that we seem to be getting an UTF-16 BOM:

![image](https://user-images.githubusercontent.com/111092/84446588-19dfb180-ac46-11ea-876d-f9a1c4937d78.png)

**Comment by udoprog** [comment]
Note that this only happens when following the README and doing the following in PowerShell:

```
cargo readme > README.md
```
Using the `-o` parameter like this gives the expected UTF-8 result.
```
cargo readme -o README.md
```

**Comment by kotauskas** [comment]
Yeah, I opened a PR (#50) sometime before on this topic. I actually solved this the same way, by replacing `>` with an `-o`.
